From f636f4a30d28d736051c829e022016d654d94c51 Mon Sep 17 00:00:00 2001
From: "Bozek, DominikX" <dominikx.bozek@intel.com>
Date: Tue, 12 Mar 2019 15:15:27 +0100
Subject: [PATCH 111/120] ASoC: Intel: Skylake: Enable codec command I/O
 functionality

Normal operation of the HDA require codec command I/O functionality
to be enabled. This patch make a simple aproach to re-enable it.

Change-Id: I0f53fe17fba1be6b5ceb0d666eb0d2a555998754
Signed-off-by: Dominik Bozek <dominikx.bozek@intel.com>
---
 sound/soc/intel/skylake/skl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index f43438a..e51ea18 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -743,6 +743,8 @@ static void skl_probe_work(struct work_struct *work)
 	list_for_each_entry(hlink, &ebus->hlink_list, list)
 		snd_hdac_ext_bus_link_put(ebus, hlink);
 
+	snd_hdac_bus_init_cmd_io(bus);
+
 	/* configure PM */
 	pm_runtime_put_noidle(bus->dev);
 	pm_runtime_allow(bus->dev);
-- 
1.9.1

